schema/data/dev/certification_url.sql

```diff
@@ -14,9 +14,6 @@ select mocks.set_mocked_time_in_transaction(
   )
 );
 
-alter table ggircs_portal.certification_url disable trigger _certification_request_email;
-alter table ggircs_portal.certification_url disable trigger _signed_by_certifier_email;
-
 with rows as (
 insert into ggircs_portal.certification_url(id, application_id, version_number)
```

Comment:
remove this?

Reply:
I figured since we're not deleting the certification_url table, and we might display historical certifications in the future, we might want to keep some dev data related to it. What do you think?